src/core/ext/filters/client_channel/lb_policy_factory.c

```diff
@@ -55,7 +55,7 @@ grpc_lb_addresses* grpc_lb_addresses_copy(const grpc_lb_addresses* addresses) {
 }
 
 void grpc_lb_addresses_set_address(grpc_lb_addresses* addresses, size_t index,
-                                   void* address, size_t address_len,
+                                   const void* address, size_t address_len,
```

Comment:
I add a `const` because I think this parameter should be read-only for a setter. Also, I made the argument [here](https://github.com/grpc/grpc/pull/12187/files#diff-8b42d287f67696893b2f8ffaae570bdfR570) a `const`.